src/store.ts

```diff
@@ -904,7 +886,7 @@ const getDataWindowFromActions = (actions: ActionData[]): DataWindow => {
 };
 
 // Get data window from actions on app load.
-const { gestures: actions } = useStore.getState();
+const { actions: actions } = useStore.getState();
```

Comment:
```suggestion
const { actions } = useStore.getState();
```

Reply:
There are others, please search for "actions: actions".

Reply:
[8f7306d](https://github.com/microbit-foundation/ml-trainer/pull/463/commits/8f7306d42de50c98a6b9a6b2d139ed1717cd7dc2)